guide/interactions/select-menus.md

```diff
@@ -68,19 +68,19 @@ Restart your bot and then send the command to a channel your bot has access to.
 You can also send message components within an ephemeral response or alongside message embeds.
 
 ```js {1,12-16,18}
-const { MessageActionRow, MessageEmbed, MessageSelectMenu } = require('discord.js');
+const { ActionRowBuilder, EmbedBuilder, SelectMenuBuilder } = require('discord.js');
 
 client.on('interactionCreate', async interaction => {
-	if (!interaction.isCommand()) return;
+	if (!interaction.isChatInputCommand()) return;
```

Comment:
I find it odd that the text at the top states to use `CommandInteraction#reply()` yet we are only allowing chat input interactions through. This will work with context menus too.